app/helpers/welcome_helper.rb

```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
```

Comment:
You don't need this helper file at this stage and chance are you'll likely not add any helper here so remove this file